applications/GeoMechanicsApplication/custom_workflows/time_loop_executor.h

```diff
@@ -1,29 +0,0 @@
-// KRATOS___
```

Comment:
This class was meant as the purely virtual interface, such that we can make a stub in one of the tests. We need to extract an interface from which both the real and dummy time_loop_executors derive

Reply:
Done